src/main/java/rx/Producer.java

```diff
@@ -47,6 +47,8 @@
      *          want the Producer to produce items at its own pace
      * @throws IllegalArgumentException if the request amount is negative
      */
-    void request(long n);
+    default void request(long n) {
```

Comment:
This is a single method interface with the main purpose of getting implemented. Having it default empty will not work with "add unimplemented methods" quick fix for once.